insonmnia/node/tokens.go

```diff
@@ -38,6 +38,22 @@ func (t *tokenAPI) Balance(ctx context.Context, _ *sonm.Empty) (*sonm.BalanceRep
 	}, nil
 }
 
+func (t *tokenAPI) Deposit(ctx context.Context, amount *sonm.BigInt) (*sonm.Empty, error) {
+	if err := t.remotes.eth.MasterchainGate().PayIn(ctx, t.remotes.key, amount.Unwrap()); err != nil {
```

Comment:
add token.approve() method here

Reply:
What value for `.approve()` is applicable here?